src/lib/rest/rest.cpp

```diff
@@ -616,6 +630,17 @@ static int contentTypeCheck(ConnectionInfo* ciP)
     return 1;
   }
 
+
+  // Case 5
+  if ((ciP->apiVersion == "v2") && (ciP->httpHeaders.contentType != "application/json"))
```

Comment:
Actually, we don't need to create an issue: is part of the #947 (I understand that the present PR doesn't implement completely #947 so we continue the work in next PRs).

I'd add a comment in #947 about it. By the moment, deffered. NTC.